README.md

```diff
@@ -19,6 +19,14 @@ to generate your site in the \_site directory, or
 
 To start Jekyll server locally.
 
+## With Docker
```

Comment:
Shouldn't this be a subheading of  `## Testing locally` above?

Reply:
This is testing/running *not* locally but in a container, so I don't think it makes sense under that header.

Reply:
right okay, then I think either of the following would actually better sense.

## Testing 

### Locally

### With Docker

---------------------

## Testing locally

## Testing with Docker

Reply:
But as I said, that header is for running the automated tests, not for running the app. If I were to move it under a top level header, I'd move it to the contributing section. Eh?

Reply:
There are no automated tests. I'm fairly sure what you added is the same running `rake server`

Reply:
Doh! You're right :) Ok, I'll put it under the testing section.